src/components/common/view/WavesBackground.native.js

```diff
@@ -1,12 +1,22 @@
-import React, { Fragment } from 'react'
+import React, { Fragment, useMemo } from 'react'
 import { StyleSheet, View } from 'react-native'
 import WavesSVG from '../../../assets/wave50.svg'
 
 export default ({ children }) => {
+  const aspectRatio = useMemo(() => {
```

Comment:
this could be moved out of the component to some util (e.g. utils/svg):

```javascript
export const getSVGAspectRatio = memoze(SvgImage => {
   // getting width and height of svg
   const { viewBox } = (<SvgImage />).props
   const [,, width, height] = viewBox.split(' ').map(Number)

   // calculate aspect ratio for svg wrapper
   return width / height
})